BackEnd/src/main/java/com/unosquare/admin_core/back_end/repository/DashboardRepository.java

```diff
@@ -47,7 +47,8 @@
                                                @Param("endDate") LocalDate endDate,
                                                @Param("today") LocalDate today);
 
-    @Query(value = "SELECT new com.unosquare.admin_core.back_end.dto.EmployeeSnapshotDto(t.teamId ,CONCAT(emp.forename, ' ', emp.surname), et.description, t.teamName) " +
+    @Query(value = "SELECT new com.unosquare.admin_core.back_end.dto.EmployeeSnapshotDto(t.teamId ," +
```

Comment:
Again this query has changed since your last pull from develop. It looks like you didn't make any code changes to this class other than formatting. It's best practice in this instant to reset it back to the original. No point in introducing unnecessary changes to a file. Adds an unnecessary history record to the file